README.md

```diff
@@ -143,6 +143,10 @@ post_install do |installer|
 end
 ```
 
+#### Manually
+1. Download and drop ```/SwiftTweaks``` folder in your project.  
```

Comment:
Hmm I don't think that's enough, right?

We haven't attached a precompiled framework to the release, so by `download`, do you mean the whole repo?